.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+npm run precommit && npm run test -- --watchAll=false && npm run build
```

Comment:
Does `npm run precommit` self-reference this file?

Reply:
No, this is so husky can execute lint-staged from those shell scripts.